ArrayAndInterface/ArrayAndInterface/Recursion/Recursion.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ArrayAndInterface.Recursion
+{
+    /// <summary>
+    /// Recursion works by defining function that calls itself until it reaches a base case.
+    /// The function is called with a set of parameters,and it returns a value.The function then
+    /// calls itself with a new set of parameters,and so on,until it reaches the base case.the
+    /// base case is a condition that stops the recursion,and it is usually defined as a simple
+    /// value or a condition that can be easily evaluated.
+    /// 
+    /// </summary>
+    /// 
+
+     public class RecursionFunction{
+      public int Calculate (int baseNum,int exponentNum)
+        {
```

Comment:
Why is there a space in the above method between method name and bracket?

Reply:
ok @AfrinShaik2000  I will Refactor it